molecule/scenario_resources/Dockerfile.j2

```diff
@@ -84,6 +67,6 @@ dnf clean all
 # Disable requiretty.
 RUN sed -i -e 's/^\(Defaults\s*requiretty\)/#--- \1/'  /etc/sudoers
 
-VOLUME ["/sys/fs/cgroup", "/tmp", "/run"]
+STOPSIGNAL SIGRTMIN+3
```

Comment:
I had to Google what this does, and I found out why we do it. But please add a comment about it.

Reply:
```suggestion
# systemd uses this as a termination signal; we want to shutdown the containers nicely...
STOPSIGNAL SIGRTMIN+3
```